astroid/exceptions.py

```diff
@@ -267,9 +267,23 @@ class ParentMissingError(AstroidError):
         target: The node for which the parent lookup failed.
```

Comment:
```suggestion
        target: The node for which the parent lookup failed.
        missing_element: The type of attribute/method that is missing because of the missing parent
```